force-app/main/triggers/classes/HOT_ServiceAppointmentHandler.cls

```diff
@@ -248,6 +264,21 @@ public without sharing class HOT_ServiceAppointmentHandler extends MyTriggers {
             ) {
                 serviceAppointmentsWithChangedAddress.add(serviceAppointment);
             }
+            if (
+                serviceAppointment.SchedStartTime != triggerOldMap.get(serviceAppointment.Id).get('SchedStartTime') ||
+                serviceAppointment.SchedEndTime != triggerOldMap.get(serviceAppointment.Id).get('SchedEndTime') &&
+                serviceAppointment.HOT_AssignedResourceId__c != null
+            ) {
```

Comment:
FSL triggere setter schedTimes til blank når tolk fjernes fra SA. Vi har logikk som setter det tilbake.
Kan dette få konsekvenser?
Skulle vi sjekket om schecStart != null && oldSchedStart != null? Samme for End?

Reply:
Åja, ja da høres det ut som en god idé. Da lager den kun wage claims når sched times endres fra/til noe annet enn null.

Reply:
Her er det jo miks av logiske operatorer || og &&
&& "regnes" jo ut først, og da blir vel dette feil. Mangler et sett med paranteser.

Reply:
Woops, 2fast